services/workflows-service/src/alert/alert.service.intg.test.ts

```
@@ -316,6 +318,145 @@
         const alerts = await prismaService.alert.findMany();
         expect(alerts).toHaveLength(0);
       });
+
+      test.only('Assigning and deciding alerts should set audit timestamps', async () => {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)